includes/civicrm.admin.php

```diff
@@ -273,6 +273,35 @@ public function run_installer() {
 
     }
 
+    $vendor_setup_paths = [];
+    $civicrm_plugin_dir_array = explode(DIRECTORY_SEPARATOR, CIVICRM_PLUGIN_DIR);
+    foreach ($civicrm_plugin_dir_array as $directory_component) {
```

Comment:
@jackrabbithanna Would it be better to apply `array_reverse( $civicrm_plugin_dir_array )` here so that directory traversal doesn't start at the root directory of the server, but starts in the CiviCRM plugin directory and works upwards instead?

Reply:
You're right, I updated the algorithm to do just that.